Dockerfile

```diff
@@ -10,11 +10,15 @@ COPY analytics/src ./src
 COPY analytics/package.json .
 COPY analytics/pnpm-lock.yaml .
 COPY analytics/tsconfig.json .
+RUN mkdir /metrics
+VOLUME ["/metrics"]
```

Comment:
Why is an anonymous volume beneficial here?

Reply:
Its not for now, removed it